src/Tribe/Commerce/Orders_Tabbed_View.php

```diff
@@ -119,6 +103,47 @@ public function render() {
 		echo $tabbed_view->render();
 	}
 
+	/**
+	 * Generates the title based on the page type and post ID.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $post_id The post ID.
+	 *
+	 * @return string The generated title.
+	 */
+	public function get_title( int $post_id ): string {
+		$page_type = tribe_get_request_var( 'page' );
+
+		// Check $page_type to confirm if we are on Order or Attendees page.
+		if ( $page_type === 'tickets-orders' ) {
+			// Translators: %1$s: the post/event title, %2$d: the post/event ID.
+			$title = _x( 'Orders for: %1$s [#%2$d]', 'orders report screen heading', 'event-tickets' );
+		} else {
+			// Translators: %1$s: the post/event title, %2$d: the post/event ID.
+			$title = _x( 'Attendees for: %1$s [#%2$d]', 'attendees report screen heading', 'event-tickets' );
+		}
+
+		$view_title = sprintf( // Translators: %1$s: the post/event title, %2$d: the post/event ID.
+			$title, get_the_title( $post_id ), $post_id );
+
+		/**
+		 * Filters whether to show the view title.
+		 *
+		 * @since 5.0.1
+		 *
+		 * @param bool $show_title Whether to show the view title.
+		 * @param int $post_id The post ID.
+		 */
+		$show_title = apply_filters( 'tribe_tickets_attendees_show_view_title', true, $post_id );
+
+		if ( ! $show_title ) {
+			$view_title = '';
+		}
```

Comment:
I feel like this should be put at the top of the `get_title()` method, then `return '';` before anything else is done, if it's `false`.